code/game/objects/items/cards_ids.dm

```diff
@@ -689,6 +691,49 @@ update_label("John Doe", "Clowny")
 	hud_state = JOB_HUD_RAWCARGO
 	access = list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MECH_MINING, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM)
 
+/obj/item/card/id/paper
+	name = "paper slip identifier"
+	desc = "Some spare papers taped into a vauge card shape, and a name scribbled on it. Seems trustworthy."
```

Comment:
Vaugue?

Reply:
wdym by vauge? sorry

Reply:
I think he meant `vauge` that looks like a typo of `vague`

Reply:
oh i didnt notice that